de.dlr.sc.virsat.model.extension.mechanical/concept/concept.concept

```diff
@@ -0,0 +1,6 @@
+Concept de.dlr.sc.virsat.model.extension.mechanical description "Concept for mechanical engineering" version 0.1
+{

```

Comment:
Also the concept should have the header 

`/*******************************************************************************
 * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
 *
 * This program and the accompanying materials are made available under the
 * terms of the Eclipse Public License 2.0 which is available at
 * http://www.eclipse.org/legal/epl-2.0.
 *
 * SPDX-License-Identifier: EPL-2.0
*******************************************************************************/`

like the other concepts.